src/broadcast_socket.cpp

```diff
@@ -276,12 +276,12 @@ namespace libtorrent
 		for (auto& s : m_unicast_sockets)
 		{
 			if (!s.socket) continue;
-			s.socket->send_to(boost::asio::buffer(buffer, size), m_multicast_endpoint, 0, e);
+			s.socket->send_to(boost::asio::buffer(buffer, std::size_t(size)), m_multicast_endpoint, 0, e);
```

Comment:
it's not possible, because `send_to` templates the first argument (`ConstBufferSequence`) and it requires to explicit the type.